exception TestError();

class TestClass() {
	void __setinvalid() {
		throw TestError();
	}
}
